src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Endpoint.cs

```diff
@@ -0,0 +1,228 @@
+        var lowerBounds = new BibleWordIdentifier(bookCodeEnum, request.StartChapter, request.StartVerse, request.StartWord).WordIdentifier;
+        var upperBounds = BibleWordIdentifier.GetUpperBoundOfWord(bookCodeEnum, request.EndChapter, request.EndVerse, request.EndWord).WordIdentifier;
+
+        var bookData = await dbContext.BibleBooks
+            .Where(bb =>
+                bb.Bible.Enabled &&
+                !bb.Bible.RestrictedLicense &&
+                bb.Bible.GreekAlignment &&
+                bb.Bible.Id == request.BibleId &&
+                bb.Code == request.BookCode.ToUpper())
+            .Select(bb => new
+            {
+                BibleId = bb.Bible.Id,
+                BibleName = bb.Bible.Name,
+                BibleAbbreviation = bb.Bible.Abbreviation,
+                BookCode = bb.Code,
+                BookName = bb.LocalizedName,
+            })
+            .FirstOrDefaultAsync(ct);
+
+        if (bookData == null)
+        {
+            await SendNotFoundAsync(ct);
+            return;
+        }
+
+        var greekAlignmentSource = await dbContext.Database
+            .SqlQueryRaw<GreekAlignmentSourceResult>(
+                GreekAlignmentSourceQuery,
+                new SqlParameter("BibleId", request.BibleId))
+            .FirstOrDefaultAsync(ct);
+
+        if (greekAlignmentSource == null)
+        {
+            await SendNotFoundAsync(ct);
+            return;
+        }
+
+        var baseQueryResults = await dbContext.Database
+            .SqlQueryRaw<BaseQueryResult>(
+                BaseQuery,
+                new SqlParameter("LowerBounds", lowerBounds),
+                new SqlParameter("UpperBounds", upperBounds),
+                new SqlParameter("BibleId", request.BibleId))
+            .ToListAsync(ct);
+
+        var greekWords = await dbContext.Database
+            .SqlQueryRaw<GreekWords>(
+                GreekWordsQuery,
+                new SqlParameter("LowerBounds", lowerBounds),
+                new SqlParameter("UpperBounds", upperBounds),
+                new SqlParameter("BibleId", request.BibleId))
+            .ToListAsync(ct);
+
+        var greekSenses = await dbContext.Database
+            .SqlQueryRaw<GreekSenses>(
+                GreekSensesQuery,
+                new SqlParameter("LowerBounds", lowerBounds),
+                new SqlParameter("UpperBounds", upperBounds),
+                new SqlParameter("BibleId", request.BibleId))
+            .ToListAsync(ct);
+
+        var response = new Response
+        {
+            BibleId = bookData.BibleId,
+            BibleName = bookData.BibleName,
+            BibleAbbreviation = bookData.BibleAbbreviation,
+            GreekBibleAbbreviation = greekAlignmentSource.Name,
+            BookCode = bookData.BookCode,
+            BookName = bookData.BookName,
+            Chapters = baseQueryResults
+                .GroupBy(r => r.BibleWordIdentifier.Chapter)
```

Comment:
The groupings here for chapter and below for verse are what motivated me to split out a bible word identifier into its components in the new `BibleWordIdentifier` class.